impl/client/src/main/javascript/web/pentaho/ccc/visual/abstract.js

```diff
@@ -1405,7 +1405,7 @@ define([
         }
 
         // Set, before, whenever lineWidth is defined.
-        var dotSize = 16;
+        var dotSize = 8;
```

Comment:
Also, the following needs to be in sync with these: 
https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js#L778

Maybe a note should also be added on both sides.